$NetBSD$

* add missing link with ossaudio library
* change to install shared artsgsl

--- flow/CMakeLists.txt.orig	2015-09-23 05:35:33.000000000 +0000
+++ flow/CMakeLists.txt
@@ -66,7 +66,7 @@ set( ${target}_SRCS
 tde_add_library( ${target} SHARED
   SOURCES ${${target}_SRCS}
   VERSION 1.0.0
-  LINK artsgsl-static artsgslpp-static artsflow_idl-shared ${AUDIOFILE_LIBRARIES} ${LIBJACK_LIBRARIES} ${LIBSNDIO_LIBRARIES}
+  LINK artsgsl-shared artsgslpp-static artsflow_idl-shared ${AUDIOFILE_LIBRARIES} ${LIBJACK_LIBRARIES} ${LIBSNDIO_LIBRARIES} ${LIBOSSAUDIO}
   DESTINATION ${LIB_INSTALL_DIR}
 )
 
